loopme-sdk/src/main/java/com/loopme/MinimizedMode.java

```diff
@@ -11,15 +11,23 @@
 public class MinimizedMode {
 
     private static final String LOG_TAG = MinimizedMode.class.getSimpleName();
-    private final AdSpotDimensions mMinimizedViewDims = new AdSpotDimensions(DEFAULT_WIDTH, DEFAULT_HEIGHT);
     private static final int DEFAULT_HEIGHT = 100;
     private static final int DEFAULT_WIDTH = 300;
-    private final int mMarginRight = 10;
-    private final int mMarginBottom = 10;
+    private final AdSpotDimensions mMinimizedViewDims = new AdSpotDimensions(DEFAULT_WIDTH, DEFAULT_HEIGHT);
     private ViewGroup mRoot;
     private RecyclerView mRecyclerView;
     private int mPosition;
 
+    public int getWidth() { return mMinimizedViewDims.getWidth(); }
+    public int getHeight() { return mMinimizedViewDims.getHeight(); }
+
+    public int getMarginRight() { return 10; }
+    public int getMarginBottom() { return 10; }
+
+    public ViewGroup getRootView() { return mRoot; }
+    public AdSpotDimensions getDimensions() { return mMinimizedViewDims; }
+    public void setPosition(int position) { mPosition = position; }
+
     public MinimizedMode(ViewGroup root, RecyclerView recyclerView) {
         if (root == null || recyclerView == null) {
```

Comment:
should we maybe consider throwing some Exception or error in case if root or recyclerView will be null

f.e.

```
public MinimizedMode(ViewGroup root, RecyclerView recyclerView) {
    if (root == null || recyclerView == null) {
        throw new IllegalArgumentException("Root view and recyclerView must not be null");
    }
    this.mRoot = root;
    this.mRecyclerView = recyclerView;
}
```

Reply:
Annotated with `@NonNull`